src/WebJobs.Script.WebHost/Diagnostics/WebHostMetricsLoggerDecorator.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.Diagnostics;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    public class WebHostMetricsLoggerDecorator : IMetricsLogger
+    {
+        private readonly IMetricsLogger _webHostMetricsLogger;
+
+        public WebHostMetricsLoggerDecorator(IMetricsLogger webHostMetricsLogger)
+        {
+            _webHostMetricsLogger = webHostMetricsLogger;
```

Comment:
Rename variable - this works against any `IMetricsLogger`, not specific to the Web Host implementation